README.md

```diff
@@ -160,6 +160,11 @@ Vault instance before starting Vault:
   - `/var/vcap/jobs/vault/tls/other_tls_cert/cert.pem`
   - `/var/vcap/jobs/vault/tls/other_tls_cert/key.pem`
 
+**WARNING** Prior to 1.0.0 release, the `VAULT_SKIP_VERIFY` environment variable is set
+if the vault address contains `https`, so connecting to
+the vault server on 127.0.0.1 (during unseal) would not throw an SSL exception. With 1.0.0
+release, the environmental variable is no longer  set  by default. The proper solution is 
+to include SAN entry of `127.0.0.1` in the vault certs.
```

Comment:
This is tricky. If you're using the default certs in the manifest of this repo you should have `127.0.0.1` as a valid alt name [per this configuration](https://github.com/cloudfoundry-community/vault-boshrelease/blob/v1.0.0/manifests/vault.yml#L59).

I'm not opposed to making sure folks get some guidance to deal with issues that come up as a result of using this Bosh release especially around things that impact security like TLS; however I take issue with two points here, curious to hear your thoughts.

1) I'm unsure about the reference to the `VAULT_SKIP_VERIFY` environment variable because as far as I know nothing in this repo was explicitly setting that value, nor has anything I'm aware of changed in 1.0 that would cause it to now be unset. I know exactly the behavior you're talking about though, which is why I think noting something about it probably is a good idea. Maybe @jhunt would know if something changed.

2) I don't agree that the proper solution is to include a SAN entry of `127.0.0.1` in the cert. I think that's really only appropriate for local work, where you might instead consider disabling TLS altogether. Take for example the case where someone is utilizing an actual purchased TLS cert. We wouldn't expect them to include a `127.0.0.1` IP SAN on it and this note could lead someone to believe that was necessary.

I have some thoughts on how to solve this in the context of the post-start script which maybe is what you're referencing when you mention `(on unseal)`. It's possible addressing that issue would make this note less pertinent. In the event we still want to note something about dealing with Vault when it's targeted by IP maybe we can just rephrase this somehow to make the advice more generically educational instead of prescriptive.